conftest.py

```
@@ -711,11 +731,16 @@ def pytest_runtest_setup(item):
-                test_name=f"{item.fspath}::{item.name}",
-                start_time=int(datetime.datetime.now().strftime("%s")),
-            )
+            # Check if module has data_collector_scope marker set to "module"
+            scope_marker = item.get_closest_marker(name="data_collector_scope")
+            if not (scope_marker and scope_marker.kwargs.get("scope") == "module"):
+                # Only record test start time if NOT using module-scoped collection
+                # (module start time is recorded in pytest_fixture_setup)
+                db = Database(base_dir=item.config.getoption("--data-collector-output-dir"))
+                db.insert_test_start_time(
+                    test_name=f"{item.fspath}::{item.name}",
+                    start_time=int(datetime.datetime.now().strftime("%s")),
```

Ruff (0.15.6)

[warning] `datetime.datetime.now()` called without a `tz` argument

(DTZ005)